tiledb/sys/src/stats.rs

```diff
@@ -0,0 +1,17 @@
+extern "C" {
+    pub fn tiledb_stats_enable() -> i32;
+
+    pub fn tiledb_stats_disable() -> i32;
+
+    pub fn tiledb_stats_reset() -> i32;
+
+    //pub fn tiledb_stats_dump(out: *mut libc::FILE) -> i32;
+
+    pub fn tiledb_stats_dump_str(out: *mut *mut ::std::ffi::c_char) -> i32;
+
+    //pub fn tiledb_stats_raw_dump(out: *mut libc::FILE) -> i32;
+
+    pub fn tiledb_stats_raw_dump_str(out: *mut *mut ::std::ffi::c_char) -> i32;
+
+    pub fn tiledb_stats_free_str(out: *mut *mut ::std::ffi::c_char) -> i32;
```

Comment:
Minor style suggestion. I'd add the note that we're intentionally not wrapping the commented out lines so folks don't think we've just not gotten to them.

```suggestion
    pub fn tiledb_stats_enable() -> i32;
    pub fn tiledb_stats_disable() -> i32;
    pub fn tiledb_stats_reset() -> i32;
    pub fn tiledb_stats_dump_str(out: *mut *mut ::std::ffi::c_char) -> i32;
    pub fn tiledb_stats_free_str(out: *mut *mut ::std::ffi::c_char) -> i32;

    // These are intentionally left unwrapped.
    //pub fn tiledb_stats_dump(out: *mut libc::FILE) -> i32;
    //pub fn tiledb_stats_raw_dump(out: *mut libc::FILE) -> i32;
    //pub fn tiledb_stats_raw_dump_str(out: *mut *mut ::std::ffi::c_char) -> i32;
```